work around gcc vs. clang:
error: addition of default argument on redeclaration makes this constructor a default constructor
see: https://stackoverflow.com/questions/18313509/default-argument-gcc-vs-clang

Index: src/srcpmessage.cpp
--- src/srcpmessage.cpp.orig
+++ src/srcpmessage.cpp
@@ -553,9 +553,9 @@ unsigned int GaMessage::getValue() const
  *   INIT <bus> GL <addr> <prot> <protv> <maxsteps> <fcount>
  *    0     1   2    3      4       5         6         7
  */
-GlMessage::GlMessage(Type type = mtGlInit, unsigned int abus = 1,
-        unsigned int addr = 1, Protocol prot = proDCC, unsigned int pv = 1,
-        unsigned int maxvs = 14, unsigned int fc = 1):
+GlMessage::GlMessage(Type type, unsigned int abus,
+        unsigned int addr, Protocol prot, unsigned int pv,
+        unsigned int maxvs, unsigned int fc):
     SrcpMessage(type, abus), address(addr), protocol(prot),
     pversion(pv), maxvsteps(maxvs), fcount(fc), fmap(0)
 {
@@ -566,10 +566,10 @@ GlMessage::GlMessage(Type type = mtGlInit, unsigned in
  *   SET <bus> GL <addr> <drivemode> <currentv> <maxv> <f1> . . <fn>
  *    0    1   2    3         4          5        6      7
  */
-GlMessage::GlMessage(Type type = mtGlInit, unsigned int abus = 1,
-        unsigned int addr = 1, unsigned int fc = 1, unsigned int dm = 1,
-        unsigned int cv = 0, unsigned int mv = 14,
-        unsigned int map = 0):
+GlMessage::GlMessage(Type type, unsigned int abus,
+        unsigned int addr, unsigned int fc, unsigned int dm,
+        unsigned int cv, unsigned int mv,
+        unsigned int map):
     SrcpMessage(type, abus), address(addr), fcount(fc), drivemode(dm),
     currentv(cv), maxv(mv), fmap(map)
 {
@@ -579,9 +579,9 @@ GlMessage::GlMessage(Type type = mtGlInit, unsigned in
  *   SET <bus> GL <addr> <drivemode> <currentv> <maxv> <f1> . . <fn>
  *    0    1   2    3         4          5        6      7
  */
-GlMessage::GlMessage(Type type = mtGlInit, unsigned int abus = 1,
-        unsigned int addr = 1, unsigned int dm = 1,
-        unsigned int cv = 0, unsigned int mv = 14, bool f0 = false):
+GlMessage::GlMessage(Type type, unsigned int abus,
+        unsigned int addr, unsigned int dm,
+        unsigned int cv, unsigned int mv, bool f0):
     SrcpMessage(type, abus), address(addr), drivemode(dm),
     currentv(cv), maxv(mv), fmap(0)
 {
@@ -773,8 +773,8 @@ void GlMessage::setFunctionMap(unsigned int map = 0)
  *   TERM <bus> GL <addr>
  *    0     1   2    3
  */
-GlMessage::GlMessage(Type type = mtGlGet, unsigned int abus = 1,
-        unsigned int addr = 1):
+GlMessage::GlMessage(Type type, unsigned int abus,
+        unsigned int addr):
     SrcpMessage(type, abus), address(addr)
 {
 }
